### ISSUE TITLE
[Bug] No const support in older browsers

### ISSUE THREAD

**Issue by pvdstel** [webpack-contrib/style-loader#339]
Occurs in v0.22.0.

**Do you want to request a *feature* or report a *bug*?**
Report a bug.

**What is the current behavior?**
`TypeError: undefined is not an object (evaluating 'e.setAttribute').`

In fc24512b395a8a3fd4074d88cd3f7b195f0bcef2, support for nonces was added. The nonce variable is declared using `const`. This `const` is causing errors in browsers not supporting it.

The Webpack-minified createStyleElement function looks as follows (basic formatting applied):

```javascript
function h(e) {
  var t = document.createElement("style");
  if (void 0 === e.attrs.type && (e.attrs.type = "text/css"), void 0 === e.attrs.nonce) {
    const t = function () { // t is reassigned
      0;
      return n.nc
    }();
    t && (e.attrs.nonce = t)
  }
  return m(t, e.attrs), // m is addAttrs
    f(e, t),
    t
}
```

It appears that `t` (`style`) is reassigned to a function. As this value is passed to `addAttrs`) it now no longer has a `setAttribute` function.

**What is the expected behavior?**

No TypeError is thrown on older browsers. Newer browsers are fine.

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**

Webpack v4.16.5
Node v8.10.0
OS: Windows 10 version 1803

**Comment by evilebottnawi** [comment]
Already fixed https://github.com/webpack-contrib/style-loader/pull/338, release will be soon

**Comment by michael-ciniawsky** [comment]
Fixed #338 and released in [`v0.22.1`](https://github.com/webpack-contrib/style-loader/releases/tag/v0.22.1) 🎉
